Files including tut header don't compile

```
err: In file included from /usr/local/include/tut/tut_exception.hpp:5:0,
                 from /usr/local/include/tut/tut.hpp:14,
                 from ../test.cpp:1:
/usr/local/include/tut/tut_result.hpp: In constructor ‘tut::test_result_posix::test_result_posix()’:
/usr/local/include/tut/tut_result.hpp:44:22: error: ‘getpid’ was not declared in this scope
         : pid(getpid())
```

--- a/include/tut/tut_console_reporter.hpp
+++ b/include/tut/tut_console_reporter.hpp
@@ -3,6 +3,12 @@
 #include <tut/tut.hpp>
 #include <cassert>
 
+#if defined(TUT_USE_POSIX)
+#include <sys/types.h>
+#include <unistd.h>
+#endif
+
+
 /**
  * Template Unit Tests Framework for C++.
  * http://tut.dozen.ru

--- a/include/tut/tut_result.hpp
+++ b/include/tut/tut_result.hpp
@@ -14,6 +14,11 @@
 #include <typeinfo>
 #endif
 
+#if defined(TUT_USE_POSIX)
+#include <sys/types.h>
+#include <unistd.h>
+#endif
+
 namespace tut
 {
 
@@ -84,7 +89,7 @@ struct test_result : public test_result_posix
     std::string name;
 
     /**
-     * result of a test 
+     * result of a test
      */
     enum result_type
     {
@@ -93,10 +98,10 @@ struct test_result : public test_result_posix
         ex,       ///< test throwed an exceptions
         warn,     ///< test finished successfully, but test destructor throwed
         term,     ///< test forced test application to terminate abnormally
-        ex_ctor,  ///< 
-        rethrown, ///< 
-        skipped,  ///< 
-        dummy     ///< 
+        ex_ctor,  ///<
+        rethrown, ///<
+        skipped,  ///<
+        dummy     ///<
     };
 
     result_type result;

--- a/include/tut/tut_xml_reporter.hpp
+++ b/include/tut/tut_xml_reporter.hpp
@@ -9,6 +9,11 @@
 #include <vector>
 #include <stdexcept>
 
+#if defined(TUT_USE_POSIX)
+#include <sys/types.h>
+#include <unistd.h>
+#endif
+
 namespace tut
 {
 

